bank_statement_journal_items/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extends the functionality of bank statements and to allow you to see journal items list.
```

Comment:
Nitpicking but I would phrase it this way: "This module adds a smart button on bank statement form view, allowing you to display its journal items"